projects/landing-page/package.json

```diff
@@ -13,8 +13,8 @@
     "prebuild": "yarn clean && mkdir dist && cp -r ./public/* ./dist/.",
     "build": "parcel build ./dist/index.html --public-url ./",
     "deep-clean": "yarn clean && rm -rf node_modules",
-    "clean": "rm -rf dist",
-    "dev": "yarn run clean && tsc --noEmit & parcel ./public/index.html",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "dev": "yarn run clean && tsc -b & parcel ./public/index.html",
```

Comment:
by removing -noEmit then "tsc" command also compiles the code instead of just typechecking. We are using parcel for that

Reply:
See above .. we can't detect changes I'd dependent projects (and therefore typecheck properly) with noEmit